i18n/translation-zh_cn.json

```diff
@@ -387,12 +390,12 @@
       "6": "星期六"
     },
     "weekdaysShort": {
-      "0": "周日",
-      "1": "周一",
-      "2": "周二",
-      "3": "周三",
+      "0": "太阳",
```

Comment:
Not sure what happened but the translations are kind of messed up now...

When you have time can you try regenerating them again? One way to do this is to copy-paste all the translation jsons in master into this branch, add the new `quality` translations, then run the script.

Reply:
Here is what I did:
1. Copied all `translation-xxx.json` file from current master branch
2. Add 3 `quality` strings to `translation-en.json` 
3. `npm run translation` to update all  `translation-xxx.json` file

After the translation strings list be build, I can see there are many updates to the existing strings, I did some check to the Chinese strings, most of the translations seem to make sense.